ekumenlabs/fruit_detection
# Basic scene configuration to launch the system.

Solves partially #4 

- Needs to fix the STL to create a hole and calibrate the sensor.
- Camera publishing on image_raw and camera_info


TODO: inject the sensor into the camera to avoid occlusion and unify prims.

To test it:

```
docker compose -f docker/docker-compose.yml --profile simulation --profile visualization build
docker compose -f docker/docker-compose.yml --profile simulation --profile visualization up
```

Then hit play in omniverse interface and see the following.

 
![vis_fruit_detection](https://github.com/user-attachments/assets/ed32eff5-62d0-4a1a-8d95-aa3f0f7ab19c)
![isaac_fruit_detection](https://github.com/user-attachments/assets/3bd8bd8e-3149-41f5-b0d0-ee9074a85f61)

Please note that you need to open a secondary viewport if you want to see the scene and the camera sensor capture.
